pom.xml

```diff
@@ -269,12 +270,17 @@
       <dependency>
         <groupId>org.ops4j.pax.url</groupId>
         <artifactId>pax-url-wrap</artifactId>
-        <version>2.6.3</version>
+        <version>${pax-url.version}</version>
       </dependency>
       <dependency>
         <groupId>org.ops4j.pax.url</groupId>
         <artifactId>pax-url-reference</artifactId>
-        <version>2.6.2</version>
+        <version>${pax-url.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.ops4j.pax.tinybundles</groupId>
+        <artifactId>tinybundles</artifactId>
+        <version>3.0.0</version>
```

Comment:
Just to make sure my understanding is correct: all the version upgrades (both in this pom as well as in BundleOptions.java) are focused only on upgrading deps for the standalone OSGi tests, right?  We're not updating anything here that would be visible to users of the Java driver (whether they use OSGi or not), correct?

Reply:
```
% mvn dependency:tree -Dincludes=org.ops4j.pax.tinybundles:tinybundles
[INFO] --- dependency:3.1.2:tree (default-cli) @ java-driver-osgi-tests ---
[INFO] com.datastax.oss:java-driver-osgi-tests:jar:4.17.1-SNAPSHOT
[INFO] \- org.ops4j.pax.exam:pax-exam-junit4:jar:4.13.4:test
[INFO]    \- org.ops4j.pax.exam:pax-exam-spi:jar:4.13.4:test
[INFO]       \- org.ops4j.pax.tinybundles:tinybundles:jar:3.0.0:test
[INFO] 
```

`tinybundles` is only used by osgi-tests under the `test` scope